### PR TITLE
[Improvement][UT] Improve Worker runner coverage

### LINKED ISSUE #15396: [Improvement][Unit Tests] Improve DolphinScheduler Worker unit tests
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

part of https://github.com/apache/dolphinscheduler/issues/10573
improve dophinscheduler-worker UT

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by pegasas**
## Purpose of the pull request

resolve: https://github.com/apache/dolphinscheduler/issues/15396

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Comment by rickchengx** [comment]
Hi, @pegasas , please fix CI

**Comment by pegasas** [comment]
> Hi, @pegasas , please fix CI

It seems there's a lot refactor after last merge.
I will fix & re-commit.

**Comment by pegasas** [comment]
> > Hi, @pegasas , please fix CI
> 
> It seems there's a lot refactor after last merge. I will fix & re-commit.

fixed.

**Comment by rickchengx** [comment]
@pegasas please check the failed CI

**Comment by pegasas** [comment]
> @pegasas please check the failed CI

image
Done.
run spotless successfully in my local

File: dolphinscheduler-worker/src/test/java/org/apache/dolphinscheduler/server/worker/runner/operator/TaskInstanceOperationFunctionTest.java
Comment (rickchengx) [inline dolphinscheduler-worker/src/test/java/org/apache/dolphinscheduler/server/worker/runner/operator/TaskInstanceOperationFunctionTest.java]: It seems that there is no test in this try block

Comment (pegasas) [reply dolphinscheduler-worker/src/test/java/org/apache/dolphinscheduler/server/worker/runner/operator/TaskInstanceOperationFunctionTest.java]: It looks like this part is no longer needed after refactor.
I've removed this part.

**Review by rickchengx — APPROVED** [review]
LGTM

**Comment by rickchengx** [comment]
will merge tomorrow if no more comments

**Pull request merged**
